ehcache-impl/src/main/java/org/ehcache/impl/internal/store/tiering/TieredStore.java

```diff
@@ -604,6 +610,11 @@ public void setInvalidationListener(final InvalidationListener<K, V> invalidatio
       // noop
     }
 
+    @Override
+    public Map<K, Store.ValueHolder<V>> bulkGetOrComputeIfAbsent(Iterable<? extends K> keys, Function<Set<? extends K>, Iterable<? extends Map.Entry<? extends K, ? extends ValueHolder<V>>>> mappingFunction) throws StoreAccessException {
+      return null;
```

Comment:
Should this return a map with the full set of keys, each with a null value?

Reply:
this should return a full set of keys, each with a null value, I have put code change for this